.github/workflows/release.yml

```
@@ -25,3 +27,68 @@ jobs:
+    outputs:
+      IMAGE_ID: ${{ steps.build.outputs.IMAGE_ID }}
+
+  deploy-alpha:
+    name: Deploy to alpha
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    needs: [build]
+    environment: staging
+    steps:
+      - env:
+          IMAGE_ID: ${{ jobs.build.ouputs.IMAGE_ID }}
```

actionlint (1.7.7)

undefined variable "jobs". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)